06.10.2025/sortedList.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include <stdbool.h>
+
+typedef struct SortedList {
+    struct ListNode* _head;
+} SortedList;
+
+/*
+ * Initialize SortedList structure.
+ */
+void sortedListInit(SortedList* pList);
+
+/*
+ * Free SortedList internals.
+ * Do not frees SortedList pointer.
```

Comment:
```suggestion
 * Does not free SortedList pointer.
```
(иногда я обращаюсь в училку английского :) )